packages/builders/src/interactions/slashCommands/SlashCommandBuilder.ts

```diff
@@ -65,6 +65,11 @@ export class SlashCommandBuilder {
 	 */
 	public readonly dm_permission: boolean | undefined = undefined;
 
+	/**
+	 * The type of this command
+	 */
+	public readonly type = 1;
```

Comment:
Also you shouldn't use magic numbers, as they aren't readable

Reply:
Alright, added the relevant type